bin/s2nd.c

```diff
@@ -399,6 +403,7 @@ int main(int argc, char *const *argv)
     int fips_mode = 0;
     int parallelize = 0;
     int use_tls13 = 0;
+    long int bytes = 0;
```

Comment:
This should be unsigned when used with strtoul.

Reply:
it is possible for strtoul() to return negative numbers, which I'm using to GUARD_EXIT() after running strtoul()

Reply:
spoke offline re: mB or gB options that could be added in the future to handle larger transactions.